Makefile

```diff
@@ -73,7 +73,7 @@ integration: submodule-init setup-ckb-test ## Run integration tests in "test" di
 
 .PHONY: integration-release
 integration-release: submodule-init setup-ckb-test prod
-	RUST_BACKTRACE=1 RUST_LOG=${INTEGRATION_RUST_LOG} test/run.sh --release -- --bin ${CARGO_TARGET_DIR}/release/ckb ${CKB_TEST_ARGS}
+	RUST_BACKTRACE=1 RUST_LOG=${INTEGRATION_RUST_LOG} test/run.sh -- --bin ${CARGO_TARGET_DIR}/release/ckb ${CKB_TEST_ARGS}
```

Comment:
should it change back?

Reply:
it can't work on my laptop with this error
```
RUST_BACKTRACE=1 RUST_LOG=info,ckb_test=debug,ckb_sync=debug,ckb_relay=debug,ckb_network=debug test/run.sh --release -- --bin /home/luo
c/rust-work/ckb-work/ckb/target/release/ckb  -c 4                                                                                      
error: The argument '--release' was provided more than once, but cannot be used multiple times
```

Reply:
got